spark/src/main/java/org/apache/iceberg/spark/source/SparkAppenderFactory.java

```diff
@@ -85,6 +72,65 @@
     this.posDeleteRowSchema = posDeleteRowSchema;
   }
 
+  public static Builder builderFor(Table table, Schema writeSchema, StructType dsSchema) {
```

Comment:
Nit: As iceberg is a library which will be embedded in user's project,  so we usually strictly control methods/classes marked as `public`.  In this case,  we don't have to mark this as `public`.  Similar comment in the following `Builder`'s  methods.

Reply:
Done